lib/util/cache/global/file.ts

```diff
@@ -7,16 +7,19 @@ function getKey(namespace: string, key: string): string {
   return `${namespace}-${key}`;
 }
 
-let renovateCache: string;
+let cacheFileName: string;
```

Comment:
This is unrelated the main refactoring but I felt it deserved a more accurate variable name